docs/detections/prebuilt-rules/rule-details/domain-added-to-google-workspace-trusted-domains.asciidoc

```diff
@@ -49,11 +49,11 @@ Trusted domains may be added by system administrators. Verify that the configura
 
 ==== Investigation guide
 
-**Config**
+*Config*
 
 The Google Workspace Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule.
 
-**Important Information Regarding Google Workspace Event Lag Times**
+ *Important Information Regarding Google Workspace Event Lag Times*
```

Comment:
See previous comment about using the NOTE admonition.

P.S. This can apply to all other similar instances in these doc updates.